nix/haskell.nix

```diff
@@ -0,0 +1,54 @@
+{ src
```

Comment:
This appears to be a new module, but it's not being imported anywhere AFAICT. I'd honestly recommend just keeping it inline in the flake anyway though, it's not a lot of code and I'm not sure if there's a significant benefit having it here in a separate module

Reply:
Made an issue for it: #127

Reply:
You can just delete the file in this PR though. It's not being imported anywhere AFAICT